toolkit/smart-contracts/offchain/src/cardano_keys.rs

```diff
@@ -41,6 +41,10 @@ impl CardanoPaymentSigningKey {
 		)
 	}
 
+	pub fn to_pub_key(&self) -> PublicKey {
```

Comment:
Minor: this `PublicKey` is from CSL, could you please reflect it in the name of method? `to_csl_pub_key`